docs/source/models/AOFDTD.rst

```diff
@@ -301,11 +324,37 @@ Solving the higher order dispersion relation for the angular frequency yields:
 
 .. math::
 
-   \omega = \frac{2}{\Delta t} \arcsin \xi\,\text{, where } \xi_\mathrm{max}
+   \omega = \frac{2}{\Delta t} \arcsin \xi\,,
+   
+where
+
+.. math::
+
+   & \xi = c\Delta t \left[ \sum\limits_{l=1/2}^{M - 1/2} (-1)^{l-\frac{1}{2}} g_l^{2M} \right] 
+
+   & \cdot \sqrt{
+      \left[
+         \frac{1}{\Delta x} \sin\left(\frac{\tilde k_x \Delta x}{2}\right)
+      \right]^2 +
+     \left[
+         \frac{1}{\Delta y} \sin\left(\frac{\tilde k_y \Delta y}{2}\right)
+     \right]^2 +
+     \left[
+         \frac{1}{\Delta z} \sin\left(\frac{\tilde k_z \Delta z}{2}\right)
+     \right]^2
+     }\,.
+
```

Comment:
As far as I remember, the expression for $\xi$ is more complicated.

Reply:
```
.. math::
   \xi = c\Delta t \sqrt{ & 
       \left[\sum\limits_{l=1/2}^{M - 1/2} g_l^{2M} \frac{\sin(\tilde k_x l \Delta x)}{\Delta x} \right]^2
       &  + \left[\sum\limits_{l=1/2}^{M - 1/2} g_l^{2M} \frac{\sin(\tilde k_y l \Delta y)}{\Delta y} \right]^2
       & + \left[\sum\limits_{l=1/2}^{M - 1/2} g_l^{2M} \frac{\sin(\tilde k_z l \Delta z)}{\Delta z} \right]^2
   }\,.
```

Reply:
Thanks for correction, you are right. Implemented your formula